reference/composedcubemaptexture.md

```diff
@@ -0,0 +1,116 @@
+## ComposedCubeMapTexture
+
+```
+ComposedCubeMapTexture {
+  vrmlField SFNode right   NULL
+  vrmlField SFNode left    NULL
+  vrmlField SFNode top     NULL
+  vrmlField SFNode bottom  NULL
+  vrmlField SFNode front   NULL
+  vrmlField SFNode back    NULL
+}
+```
+
+### Description
+
+The [ComposedCubeMapTexture](#composedcubemaptexture) node defines a cubic texture used for environment mapping.
+
+Each of the six fields takes a texture corresponding to the side of the environment. For instance `right` takes the texture displayed on the side of the cube along the +X direction. Accordingly we have the following order, with the Z-axis pointing outwards: right:+X (0), left:-X (1), top:+Y (2), bottom:-Y (3), front:+Z (4), back:-Z (5).
```

Comment:
Is "the Z-axis being the result of the cross-poduct of Y by X" brief and clear enough?